src/main/resources/data/minecraft/tags/blocks/leaves.json

```diff
@@ -1,11 +1,6 @@
 {
+  "replace": false,
```

Comment:
Replace is always false by default, so technically unnecessary, but not bad to have.